task3/Ex6.Rmd

```diff
@@ -134,9 +138,76 @@ As the average day temperature is the most intuitive parameter, it was chosen fo
 ##Does the spatial aggregation to districts change the overall picture?
 The overall picture is definitely changed by the spatial aggregation. The amount of hobos per district varies as well as the area of the different districts. Especially in large suburban district with few hobos, one outlier can dominate a large area. On the other hand, a lot of measurements took place in smaller districts in the city center, which might therefore be more reliable but only represent a small area.
 
-# cleanup
+#Maps of other indices:
+#mean night map
 ```{r}
-dbDisconnect(con)
+class_breaks2 <- c(floor(seq(min(hobo_inter$mean_night), max(hobo_inter$mean_night), length.out = 8)),12)
+tm_shape(fr_dis_indi_sf) +
+  tm_fill("mean_night", style = "cont", breaks = class_breaks2, legend.show = F) +
+  tm_borders() +
+    tm_text("name", just = "center", size = 0.7, col = "black") +
+tm_shape(hobo_inter) +
+  tm_dots(col = "mean_night", shape = 21, size = 0.4, style = "cont", 
+          breaks = class_breaks2, title = "Temp. [°C]") +

```

Comment:
I would add "midpoint = NA" after the title argument. Then the temperatures are easier to distinguish.